INSTALL.md

```diff
@@ -60,24 +60,46 @@ You will need:
  * PhysFS
  * oggenc (to generate music files)
 
-On Ubuntu (and probably any other Debian-based system), you can use the following command to install all required packages:
+On Ubuntu (and probably any other Debian-based system) you can use the following commands.
+
+The apt-get update is optional but recommended to use before install packages:
+
+```
+  $ sudo apt-get update
+```
+
+Use the following command to install all required packages:
 ```
-  $ apt-get install build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
+  $ sudo apt-get install --no-install-recommends build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
 ```
+key `--no-install-recommends` is optional.
 
 Make sure you install the packages along with header files (often distributed in separate *-dev packages). If you miss any requirements,
 CMake should warn you.
 
-To compile colobot, run your favorite shell and download colobot source files:
+Create a folder to sources. And enter into it:
+
 ```
-  $ git clone https://github.com/colobot/colobot.git /path/to/colobot/sources
-  $ cd /path/to/colobot/sources
-  $ git submodule update --init
+  mkdir colobot_sources
```

Comment:
Is there a point to make a double directory in there? Git automatically clones everything into `colobot` subdirectory of `some_folder`, unless you tell it to do otherwise, so I'm not sure why we're moving from `some_folder/colobot/<files>` to `some_folder/colobot/colobot/<files>`.
I think it would be better to leave base command as `$ git clone https://github.com/colobot/colobot.git`, with optional command `$ git clone https://github.com/colobot/colobot.git /path/to/colobot/sources` if someone wants to clone it somewhere else. Unless there is a good reason to have duplicated root directory.

Reply:
I'm not sure if it was actually needed to change following blocks. Was there something particularly wrong with them?

Reply:
Is there a point to make a double directory in there?
It confuses me too. But if not enter to folder `cd colobot`.  `git submodule update --init` is not runned.